geometry_msgs/Point[] clst_centers
float64[] clst_radius
